src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -65,16 +65,17 @@ public static class TextCatalog
                 outputColumnName, inputColumnNames, options);
 
         /// <summary>
-        /// Create a <see cref="TokenizingByCharactersEstimator"/>, which tokenizes characters by splitting text into sequences of characters
+        /// Create a <see cref="TokenizingByCharactersEstimator"/>, which tokenizes by splitting text into sequences of characters
         /// using a sliding window.
         /// </summary>
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
-        /// This column's data type will be a variable-sized vector of keys.</param>
+        /// This column's data type will be a variable-sized vector of <see cref="System.UInt16"/>.</param>
```

Comment:
><see cref="System.UInt16"/>. [](start = 71, length = 28)

TokenizungByCharacters.cs
line 631:
`result[colInfo.outputColumnName] = new SchemaShape.Column(colInfo.outputColumnName, SchemaShape.Column.VectorKind.VariableVector, NumberDataViewType.UInt16, true, new SchemaShape(metadata.ToArray()));`

that `true` after means it's gonna be key type with implementation type of Uint16.